src/App.css

```diff
@@ -3,3 +3,67 @@
   padding: 0;
   box-sizing: border-box;
 }
+
+nav {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  color: black;
+  background-color: #e9e9e9;
+  font-size: 20px;
+  margin-bottom: 25px;
+}
+:visited,
+:link {
+  text-decoration: none;
+  color: inherit;
+}
+
+.logo {
+  position: relative;
+  left: 0px;
+  top: 0px;
+}
+.rest {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.rest > * {
+  margin-left: 15px;
+}
+.active {
+  color: rgb(189, 83, 83);
```

Comment:
decide on a standard, use hex colors(#) by default and introduce rgba when dealing with opacity